src/main/java/com/additional/library/domain/Rent.java

```diff
@@ -18,16 +20,18 @@
     private Reader reader;
     private Exemplar exemplar;
 
-    public Rent() {
-        this.rentDate = LocalDate.now();
+    public Rent(LocalDate rentDate, Reader reader, Exemplar exemplar) {
+        this.rentDate = rentDate;
+        this.reader = reader;
+        this.exemplar = exemplar;
         this.returnDate = rentDate.plusDays(28);
```

Comment:
czemu już ustawiasz datę zwrotu? Moim zdaniem to oznacza, że egzemplarz został już zwrócony :/